### PR TITLE
add 'window(focus=True)'

### PR DESCRIPTION
**Pull request by mi4code**
#1030

### CODE REVIEW

File: webview/platforms/winforms.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 from webview.window import FixPoint
 from webview.menu import Menu, MenuAction, MenuSeparator
 
+import win32con, win32gui
```
[the remark this answers is not in the document]
Comment (mi4code) [reply webview/platforms/winforms.py]: Ok, I have just replaced it with the `windll.user32`

**Comment by r0x0r** [comment]
What is situation with Cocoa?

**Comment by mi4code** [comment]
Sorry, I have no idea how to do this with cocoa. I tried to find something (like [this](https://developer.apple.com/documentation/appkit/nswindowstylemask/nswindowstylemasknonactivatingpanel) or [this](https://stackoverflow.com/questions/11936915/cocoa-prevent-window-activation)) but I dont have mac to try to do something with that. Do you mind adding this without cocoa support?

**Comment by r0x0r** [comment]
Cocoa is tricky, let's do this without it at this point. I will see if I can make it work. Thanks for the links.

**Pull request merged**